Make requires libtalloc-dev on Ubuntu 16.04

On a fresh Ubuntu 16.04 machine `sudo apt-get install libtalloc-dev` was required prior to being able to complete the make process successfully.

_Make error without libtalloc-dev installed:_
```linux/includes.h:30:20: fatal error: talloc.h: No such file or directory
compilation terminated.
<builtin>: recipe for target 'cgi.o' failed
make: *** [cgi.o] Error 1
```

--- a/README.md
+++ b/README.md
@@ -16,3 +16,8 @@ Some information on the JSON protocol used is here:
 
  https://docs.google.com/document/d/12IQFXDRIif06BiriHSCGdiJGZ6zsQ_phQsG_iI6_MAo/edit?usp=sharing
  
+ #### Build Notes
+ _Ubuntu 16.04_
+ - libtalloc-dev is required to build this project. You can install the required package with the following command: `sudo apt-get install libtalloc-dev`
+ 
+ 

